styles.css

```diff
@@ -9,721 +11,847 @@ body {
+    background-color: #fff;
+    border-radius: 0 70px 0 0;
+  }
+
+  .about {
+    margin-bottom: 0.938rem;
+    margin-top: 3.125rem;
+    margin-left: 0;
+    font-size: 2.5rem;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 900;
+    color: #172b4d;
+  }
+
+  .experience {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1.188rem;
+    font-weight: thin;
+    margin-bottom: 0.625rem;
+    color: #344563;
+  }
+
+  .connect-1 {
+    height: 3.125rem;
+    color: #7f8cff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 26px;
+  }
+
+  .icons-1 {
+    height: 3.125rem;
+    display: flex;
+    justify-content: space-evenly;
+    list-style-type: none;
+    margin-right: 5rem;
+  }
+
+  .icons-2 {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .resume {
+    margin-bottom: 1.125rem;
+    border-radius: 0.5rem;
+    font-size: 1.188rem;
+    width: 9.375rem;
+    height: 3.125rem;
+    border: 0.063rem solid blue;
+    background-color: #fff;
+    color: #396df2;
+  }
+
+  .resume:hover {
+    background-color: #afc1f0;
+  }
+
+  .languages-1 {
+    font-size: 23px;
+    color: #000;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    margin-bottom: 25px;
+  }
+
+  #languages-23 {
+    background-image: url('images/Union-down.svg');
+    background-repeat: no-repeat;
+    background-position: 18rem;
+  }
+
+  .language-icons {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    gap: 12px;
+  }
+
+  .language-icons li {
+    display: flex;
+    align-items: center;
+    background-color: #f7f7f9;
+    padding: 9px 12px;
+    border-radius: 12px;
+  }
+
+  .language-icons li p {
+    margin-left: 16px;
+  }
+
+  .alternative {
+    width: 70vw;
+  }
+
+  .language-container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .languages-below {
+    list-style-type: none;
+  }
+
+  .languages-below li {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 88%;
+  }
+
+  .frame {
+    font-family: 'Poppins', sans-serif;
+    margin-top: 35px;
+    font-size: 1.563rem;
+    font-weight: 500;
+    height: 2.5rem;
+    border-bottom: 0.063rem solid rgb(186, 183, 183);
+  }
+
+  /* #frame-1 {
+    background-image: url('images/Union.svg');
+    background-repeat: no-repeat;
+    background-position: 18rem;
+  }
+
+  #skill-1 {
+    background-image: url('images/Union.svg');
+    background-repeat: no-repeat;
+    background-position: 18rem;
+  } */
+
+  .skills {
+    font-family: 'Poppins', sans-serif;
+    margin-top: 15px;
+    font-size: 1.563rem;
+    font-weight: 500;
+    height: 3.125rem;
+    margin-bottom: 0.938rem;
+    border-bottom: 0.063rem solid rgb(186, 183, 183);
+  }
+
+  .contact-form {
+    background: url('images/contact\ form\ shapes-mobile.svg') no-repeat;
+    background-size: auto;
```

Comment:
- [ ] dear @MarkYegon7, Just as mentioned by the previous reviewer, we need to center out form content appropriately, currently, your button and title are not aligned to the center as shown on the figma design. This is just to make sure we have a correct layout as given in the figma design. 

- [Optional] Please try to add the border-radius at the top of the from. As frontend engineers, we are expected to deliver something identical to the original design. Also, consider minimizing the width for your form elements as shown in the figma design. 
### Your design 
![image](https://github.com/MarkYegon7/Portfolio-Setup/assets/69077061/bf278cb4-e3e0-490e-91f6-83207e2cc76f)
### Figma design 
![image](https://github.com/MarkYegon7/Portfolio-Setup/assets/69077061/feb64fe2-e597-4982-bfa7-a056c58ac526)